tarantool/connection.py

```diff
@@ -134,7 +136,7 @@ def close(self):
         self._socket = None
 
     def connect_basic(self):
-        if self.host == None:
+        if self.socket:
```

Comment:
Why're you doing this incompatible changes? use separate PR's for them.

Reply:
Sorry, I'll send another PR for this. I thought this repo no longer active